site/gatsby-site/pages/apps/submit.js

```diff
@@ -96,214 +98,240 @@ const dateRegExp = /^(\+?\d{4})?\s?-?\s?(\(?\d{2}\)?)\s?-?\s?(\(?\d{2}\)?)\s?-?\
+  'Text of the article or transcript of the rich media',
   `Copy, paste, and validate the textual contents of the incident report
   match the contents of the URL linked above. Please scrub it of all advertisements
   and other media that should not be indexed and searched when users are discovering
-  incidents in the database.`)
+  incidents in the database.`
 );
 
-const RelatedLayout = ({related}) => {
-  if(related.length < 1) {
-    return (<></>)
+const RelatedLayout = ({ related }) => {
+  if (related.length < 1) {
+    return <></>;
   }
   return (
     <>
       <p />
       <ListGroup>
-        <ListGroup.Item key={"header"}>
+        <ListGroup.Item key={'header'}>
           The following incident reports exist for the incident you are reporting on
         </ListGroup.Item>
-        {related.map((val) =>
-          <ListGroup.Item key={val["url"]}>
-              <a href={val["url"]} target="_blank">{val["title"]}</a>
+        {related.map(val => (
+          <ListGroup.Item key={val['url']}>
+            <a href={val['url']} target="_blank" rel="noreferrer">
+              {val['title']}
+            </a>
           </ListGroup.Item>
-        )}
+        ))}
       </ListGroup>
     </>
-  )
-}
+  );
+};
 
 export default class SubmitForm extends Component {
-
   constructor(props) {
-    super(props)
-    const defaultValues = [{data:{
-      authors:"", submitters:"Anonymous", title:"",
-      text:"", url:"", incident_date:"", date_published:"",
-      date_downloaded:"", image_url:"", incident_id:""
-    }}]
+    super(props);
+    const defaultValues = [
+      {
+        data: {
+          authors: '',
+          submitters: 'Anonymous',
+          title: '',
+          text: '',
+          url: '',
+          incident_date: '',
+          date_published: '',
+          date_downloaded: '',
+          image_url: '',
+          incident_id: '',
+        },
+      },
+    ];
+
     this.state = {
       newIncidentData: {},
       admin: false,
       csvData: defaultValues,
       csvIndex: 0,
-      relatedIncidents: []
-    }
+      relatedIncidents: [],
+    };
   }
 
   componentDidMount() {
     const that = this;
-    getUser().then(function(user){
-      that.setState((state, props) => {
-        return {admin: user.type=="token"};
-      });
+
+    getUser().then(function(user) {
+      that.setState({ admin: user.type == 'token' });
     });
   }
 
-  handleOnDrop = (data) => {
-    console.log('---------------------------')
-    console.log(data)
-    this.setState({csvData: data})
-    console.log('---------------------------')
-  }
+  handleOnDrop = data => {
+    console.log('---------------------------');
+    console.log(data);
+    this.setState({ csvData: data });
+    console.log('---------------------------');
+  };
 
   handleOnError = (err, file, inputElem, reason) => {
-    console.log(err)
-  }
+    console.log(err, file, inputElem, reason);
+  };
 
-  updateRelated = (incident_id) => {
```

Comment:
parens were added in another case above. Are parens needed or not needed per the style guide?

Reply:
https://prettier.io/docs/en/options.html#arrow-function-parentheses

We did not overwritten any rules, so it must be using always.

Reply:
Why this change then? Should it not have parens around `incident_id`?

Reply:
This is my only lingering doubt about this merge request, but I think I am good with landing this as soon as we close out this discussion.

Reply:
It's our prettier version, "prettier": "^1.19.1", which is using "avoid" as default. shall we upgrade package version to latest?

Reply:
We might as well be at the latest since changing this again at a later time will be annoying. Regardless, I don't feel strongly about it so long as we are consistent. Would you like to proceed as-is?

Reply:
I've updated the package versions and fixed all lint issues.